clientd3d/draw3d.c

```diff
@@ -387,8 +387,8 @@ void UpdateRoom3D(room_type *room, Draw3DParams *params)
 
    // Size of offscreen bitmap.
    area.x = area.y = 0;
-   area.cx = main_viewport_width;
-   area.cy = main_viewport_height;
+   area.cx = CLASSIC_WIDTH * 1.5;
```

Comment:
We have a lot of magic numbers like this in the renderer now.  Is it possible to derive this from other numbers?  Or at least document why 1.5 and not some other value.

Reply:
Also I'll add that the comment says that this is the size of the offscreen bitmap, not the size of a viewport used for culling.  Are you sure you're not affecting the rendered resolution with this change?  I would expect the offscreen bitmap to be the same size as the viewport, especially for software mode.